src/coreComponents/physicsSolvers/contact/SolidMechanicsEmbeddedFractures.cpp

```diff
@@ -77,7 +78,7 @@ void SolidMechanicsEmbeddedFractures::postProcessInput()
   {
     linParams.mgr.strategy = LinearSolverParameters::MGR::StrategyType::solidMechanicsEmbeddedFractures;
     linParams.mgr.separateComponents = true;
-    linParams.mgr.displacementFieldName = keys::TotalDisplacement;
+    linParams.mgr.displacementFieldName = extrinsicMeshData::solidMechanics::totalDisplacement::key();
```

Comment:
This is kind of long. Would anyone be opposed to using `using namespace extrinsicMeshData` in source  files?

Reply:
We could be considering renaming `extrinsicMeshData` to something like `fields`?
This would lead to things like
```c++
arrayView1d< real64 const > const & p = subregion.getField< fields::well::pressure >();
```
and
```
nodeMgr.registerField< fields::solidMechanics::totalDisplacement >();
```
which I find very clarifying (for new comers, but for old folks too!). (eventually, we should be declaring on quadratures, but it's another question 🤣 )
We could rename `EXTRINSIC_MESH_DATA_TRAIT` into `DECLARE_FIELD` or `DECLARE_FIELD_INFO` as well?

Reply:
I think it would be fine to do this.

Reply:
Done, I added `using namespace extrinsicMeshData` at the top of the cpp files that I touched in this PR. In the next PR we can do what @TotoGaz suggests